Assets/Character/AaronTestAssets/RollState.cs

```diff
@@ -9,7 +9,7 @@ public class RollState : PlayerState {
     private Animator an;
     private Player play;
     private Rigidbody2D rgb;
-    private float timer = 0.2f;
+    private float timer = 0.12f;
```

Comment:
Make these configurable via Inspector (make them public so they show up in unity interface). You can also look at this https://docs.unity3d.com/ScriptReference/SerializeField.html if you wanna keep it private